radar_parlamentar/plenaria/ordenacao.py

```diff
@@ -91,7 +91,7 @@ def ordenar_partidos(casa, periodo):
     analise_periodo = analisador.analisa()
     coords = analise_periodo.coordenadas_partidos
     coords_list = []
-    for k in coords.iteritems():
+    for k in list(coords.items()):
```

Comment:
esse `list()` não é necessário, vocês podem fazer
```
for coord in coords.items():
    coords_list.append(coord)
```
ou, se o objetivo é somente transformar o dicionário numa lista, pode-se fazer simplesmente
`coords_list = list(coords.items())`

ou ignorar da linha 93 até a 95 e mudar a linha 96 para:

`partidos = sorted([(x,y[0] for x, y in coords.items()], key=itemgetter(1,0))`

Isso se eu entendi direito a lógica e o que tem dentro de cada variável... já não lembro direito rs. Um "sample" ajuda a entender melhor.